packages/editor/src/editor/FormToolbar.tsx

```diff
@@ -98,9 +99,11 @@ export const FormToolbar = forwardRef<HTMLDivElement>((_, ref) => {
           <PalettePopover label='Composite' icon={IvyIcons.File}>
             <CompositePalette />
           </PalettePopover>
-          <PalettePopover label='Data' icon={IvyIcons.DatabaseLink}>
-            <DataClassPalette />
-          </PalettePopover>
+          <PaletteButton text='Data'>
+            <DataClassDialog worfkflowButtonsInit={false}>
+              <Button icon={IvyIcons.DatabaseLink} size='large' aria-label='CreateFromData' />
```

Comment:
aria-labels are for humans (or better content reader for accessilibty), so I think if you add a aria-label (which is good IMO) then you should add it in normal words: 'Create from data'